inspire/modules/deposit/views.py

```diff
@@ -25,3 +41,81 @@
     template_folder='templates',
     static_folder="static",
 )
+
+
+chart_types = {'column': 'Column',
```

Comment:
``blueprint`` is a special case but if you define a global variable the convention is to use capital letters.